applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```diff
@@ -14,46 +26,90 @@ def __init__(self, Model, settings ):
             "reference_area": 1.0,
+            "moment_reference_point" : [],
             "create_output_file": false
         }''')
 
         settings.ValidateAndAssignDefaults(default_parameters)
 
         self.body_model_part = Model[settings["model_part_name"].GetString()]
-        self.velocity_infinity = [0,0,0]
-        self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
-        self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
-        self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
+        self.fluid_model_part = self.body_model_part.GetRootModelPart()
         self.reference_area =  settings["reference_area"].GetDouble()
         self.create_output_file = settings["create_output_file"].GetBool()
+        self.moment_reference_point = settings["moment_reference_point"].GetVector()
+        
 
     def ExecuteFinalizeSolutionStep(self):
         KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','COMPUTE LIFT')
 
-        rx = 0.0
-        ry = 0.0
-        rz = 0.0
+        total_force = KratosMultiphysics.Vector(3)
+        total_moment = KratosMultiphysics.Vector(3)
 
         for cond in self.body_model_part.Conditions:
             n = cond.GetValue(KratosMultiphysics.NORMAL)
             cp = cond.GetValue(KratosMultiphysics.PRESSURE)
 
-            rx += n[0]*cp
-            ry += n[1]*cp
-            rz += n[2]*cp
+            # Computing forces
+            total_force += n.__mul__(cp)
+
+            # Computing moment
+            mid_point = cond.GetGeometry().Center()
+            lever = mid_point-self.moment_reference_point
+            total_moment += CrossProduct(lever, n*(-cp))
+
+        force_coefficient = total_force.__div__(self.reference_area)
+        self.moment_coefficient = total_moment[2]/self.reference_area
+
+        self.__ReadWakeDirection()
 
-        RZ = rz/self.reference_area
-        RX = rx/self.reference_area
-        RY = ry/self.reference_area
+        self.lift_coefficient = DotProduct(force_coefficient,self.wake_normal)
+        self.Cd = DotProduct(force_coefficient,self.wake_direction)
 
-        Cl = RY
-        Cd = RX
+        self.__ComputeLiftJump()
 
-        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','Cl = ', Cl)
-        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','Cd = ', Cd)
-        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','RZ = ', RZ)
-        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess','Mach = ', self.velocity_infinity[0]/340)
+        KratosMultiphysics.Logger.PrintInfo(' Cl = ', self.lift_coefficient)
+        KratosMultiphysics.Logger.PrintInfo(' Cd = ', self.Cd)
+        KratosMultiphysics.Logger.PrintInfo(' RZ = ', force_coefficient[2])
+        KratosMultiphysics.Logger.PrintInfo(' Cm = ', self.moment_coefficient)
+        KratosMultiphysics.Logger.PrintInfo(' Cl = ' , self.lift_coefficient_jump, ' = 2 * DPhi / U_inf ')
```

Comment:
```suggestion
        KratosMultiphysics.Logger.PrintInfo('ComputeLiftProcess',' Cl = ' , self.lift_coefficient_jump, ' = 2 * DPhi / U_inf ')
```
also maybe better to use the long names?